README.md

```diff
@@ -43,7 +43,7 @@ See Example below
 
 ## Example
 
-    $ firebase-import --database_url https://test.firebaseio-demo.com --path / --json test.json --service_account /path/to/your/service_account.json
+    $ firebase-import --database_url https://test.firebaseio-demo.com --path // --json test.json --service_account /path/to/your/service_account.json
```

Comment:
Hrm.  Windows doesn't like `/` as an argument?  `//` looks strange to me and I'm not sure it'll work on non-Windows.  Well, I guess it will but only because Firebase is forgiving of empty path segments.

Could you do `"/"` instead?